pkg/report/darwin.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package report
+
+import (
+	"regexp"
+)
+
+func ctorDarwin(cfg *config) (Reporter, []string, error) {
+	symbolizeRes := []*regexp.Regexp{}
+	ctx, err := ctorBSD(cfg, darwinOopses, symbolizeRes)
+	return ctx, nil, err
+}
+
+var darwinOopses = append([]*oops{
+	{
+		[]byte("panic(cpu "),
+		[]oopsFormat{
+			{
+				title: compile("panic\\(.+\\): (assertion failed: .*), file"),
+				fmt:   "panic: %[1]v",
+			},
+			{
+				title: compile("panic\\(.+\\): Kernel trap at 0x[0-9a-f]+, (type .*), registers"),
+				fmt:   "panic: Kernel trap %[1]v",
+			},
+			{
+				title: compile("panic\\(.+\\): \"in6p_route_copyout: wrong or corrupted route:"),
```

Comment:
Do you have any sample reports? If yes, please add them in pkg/report/testdata/darwin/report/* They are super useful.

Also these in6p_route_copyout look very specific. Is it just copy-paste from some BSD? Or you actually got these crashes? Each real pattern needs at least 1 test report.

Reply:
They are all real panics I saw on XNU while testing.

Reply:
Done, I hope what I added makes sense like that?